Update the mission (#161)
alchemistry/alchemlyb: docs/index.rst

=========================================

**alchemlyb** is a library for doing alchemical free energy calculations more easily.
It includes functions for parsing data from formats common to existing MD engines, subsampling these data, and fitting these data with an estimator to obtain free energies.

> These functions are simple in usage and pure in scope, and can be chained together to build customized analyses of data.

Reviewer: Don't delete — this is still true.

Expand the scope in the next paragraph.

> It seeks to provide fexlible building blocks which could be chained together for customised analysis and workflows that use these building bocks to allow foolproof automatic analysis.

Reviewer: flexible

Instead of "foolproof" (because if you build foolproof tools, the universe builds a better fool — and also because it's not nice to refer to users as "fools") use "robust" and "general". These larger building blocks can also be seen as reference implementations/examples. They have to follow best practices. Remove the "automatic" — that just means different things to different people.

It seeks to provide fexlible building blocks which could be chained together for customised analysis and workflows that use these building bocks to allow foolproof automatic analysis.

> **alchemlyb** seeks to be as boring and simple as possible to enable more complex work.

Reviewer: leave it, that's part of the brand

> Different building blocks covers functions for parsing data from formats common to existing MD engines, subsampling these data, fitting these data with an estimator to obtain free energies, and plotting the results.

Reviewer: can be added

=========================================

**alchemlyb** is a library for doing alchemical free energy calculations more easily.

It seeks to provide flexible building blocks covering functions for parsing data from formats common to existing MD engines, subsampling these data, fitting these data with an estimator to obtain free energies, and plotting the results.

> These functions are simple in usage and pure in scope, and can be chained together to build customized analyses of data. General and robust workflows following the best practice are also provided that can be seen as reference implementations and examples.

Reviewer: ```suggestion
These functions are simple in usage and pure in scope, and can be chained together to build customized analyses of data. General and robust workflows following best practices are also provided, which can be used as reference implementations and examples.
```
